test/unit/definitions/divorce/CaseEventToFields.test.js

```diff
@@ -98,7 +101,8 @@ describe('CaseEventToFields (non-prod)', () => {
       'CaseField-alt-service-process-server-nonprod',
       'CaseField-deemed-and-dispensed-nonprod',
       'CaseField-general-email-nonprod',
-      'CaseField-general-referral-nonprod'
+      'CaseField-general-referral-nonprod',
+      'CaseField-alternative-service-nonprod'
```

Comment:
Shouldn't you have both alternative service types here?

Reply:
For alternative service its just `ServedByAlternativeMethod` being set to `Yes` or No` unlike Service Applications where we capture if its `Deemed` or `Dispensed`

Reply:
Both are there, I think the other one is at the top :)

Reply:
please try and keep the alphabetical order :)

Reply:
@yarpo, names reorder.